src/query/api/v1/handler/prometheus/common.go

```diff
@@ -104,11 +106,14 @@ func ParseRequestTimeout(r *http.Request, configFetchTimeout time.Duration) (tim
 	return duration, nil
 }
 
-// ParseTagCompletionParamsToQuery parses all params from the GET request
+// ParseTagCompletionParamsToQuery parses all params from the GET request.
 func ParseTagCompletionParamsToQuery(
 	r *http.Request,
 ) (*storage.CompleteTagsQuery, *xhttp.ParseError) {
-	tagQuery := storage.CompleteTagsQuery{}
+	tagQuery := storage.CompleteTagsQuery{
+		Start: time.Time{},
```

Comment:
just making sure this is the default that can be overriden by the parameters passed over http

Reply:
It can not... fixing